server/base/src/main/java/org/apache/accumulo/server/fs/VolumeManager.java

```diff
@@ -149,6 +150,11 @@ FSDataOutputStream createSyncable(Path logPath, int buffersize, short replicatio
   // volumes
   boolean rename(Path path, Path newPath) throws IOException;
 
+  // rename lots of files at once in a thread pool
+  // returns once all the threads have completed
```

Comment:
could add something to the docs about this operation being idempotent, safe to call again after previous partial run of bulkRename